python/adapters.py

```diff
@@ -49,7 +49,8 @@ def _encode(self, obj, context, path=None):
         return round(obj.timestamp())
 
     def _decode(self, obj, context, path=None):
-        return datetime.datetime.utcfromtimestamp(obj)
+        t = datetime.datetime.fromtimestamp(obj, datetime.timezone.utc)
+        return t.replace(tzinfo=None)
```

Comment:
I think it would be fine to return here a timezone-aware timestamp with UTC timezone. Even though it's changing the API of this function, I don't think anyone is using the output of this function for anything other than printing or formatting the timestamp in human-readable format.